client/src/App/App.css

```diff
@@ -1,4 +1,5 @@
 @import '~antd/dist/antd.css';
+@import '~bootstrap/dist/css/bootstrap.css';
```

Comment:
Why do we additionally need bootstrap?

Reply:
its not additional . i cant find import of bootstrap any where else 🤔 in project and bootstrap classes werent working without this import

Reply:
I meant in addition to Antd.  Why do we also need Bootstrap?  I thought that we are just using Antd.

cc: @kelvin-muchiri @dubdabasoduba

Reply:
@moshthepitt Seems like there is mix up usage of the two. For instance, [this component](https://github.com/OpenSRP/web/blob/master/packages/keycloak-user-management/src/forms/index.tsx) uses both Formik and Form from and, without the import above, (as it is currently on master), there is no styling on that form apart from antd form elements
![Screenshot from 2020-10-23 14-42-57](https://user-images.githubusercontent.com/19818819/97000060-dde72200-153e-11eb-9984-a48fd3641a25.png)

Reply:
@moshthepitt In the above case and any other case if it exists, should a refactor be made to only use antd components and layout?

Reply:
@kelvin-muchiri yes, absolutely.  We need to use antd everywhere.  And for forms we are using [antd-formik](https://github.com/jannikbuschke/formik-antd).  I do not anticipate a reason to use Bootstrap anywhere.  

cc: @p-netm @ashfahan @HaseemUlHaq

Reply:
I am removing formik and refactoring to used antd for all form elements in the user details form

Reply:
@kelvin-muchiri no we agreed that we are using formik via antd-formik

Reply:
@moshthepitt what should we do about spacing and paddings and other front end utility classes ?

should we use custom css styles everywhere or just make a utility css and use it globally ?

Reply:
> what should we do about spacing and paddings and other front end utility classes ?
> should we use custom css styles everywhere or just make a utility css and use it globally ?

@ashfahan @kelvin-muchiri  does antd not offer any of that?

Reply:
@moshthepitt i dont think it does offer those type of classes .

even at their own site they are writing custom styles to do the styling

Reply:
Ugh okay.  Here are my quick thoughts:

- Including the entire bootstrap for just some small parts is not cool.  If we can get just the css we need then its fine
- If whatever antd use can be taken an re-used by us then its also fine

@kelvin-muchiri @dubdabasoduba I suggest that we discuss this during the next standup and come to an agreement on how we will deal with this henceforth

Reply:
> no we agreed that we are using formik via antd-formik

@moshthepitt Is this a package?

Reply:
@kelvin-muchiri yes https://github.com/jannikbuschke/formik-antd

Reply:
Never mind, I guess you mean `formik-antd`, I followed the link above

Reply:
more conversation on  #105